py/nightwatch/webpages/tables.py

```diff
@@ -22,8 +22,11 @@ def write_nights_table(outfile, exposures):
         exposures: table with columns NIGHT, EXPID
     Returns: HTML file written to outfile path
     """    
-    env = jinja2.Environment(
-        loader=jinja2.PackageLoader('nightwatch.webpages', 'templates')
+   env = jinja2.Environment(
```

Comment:
@ana-lyons ditto for this indentation; off by 1?

Reply:
Fixed!